pom.xml

```diff
@@ -103,6 +102,8 @@
 		<module>artemis-test</module>
 		<module>artemis-cli</module>
 		<module>artemis-gwt</module>
+		<module>artemis-serializer-backends/artemis-json</module>
+		<module>artemis-serializer-backends/artemis-json-libgdx</module>
   </modules>
```

Comment:
Should we put a pom in artemis-serializer-backends, let it build its child modules?